frontend/src/features/mapSideBar/SidebarContents/subforms/AssociatedLandReviewPage.tsx

```diff
@@ -115,9 +116,10 @@ export const AssociatedLandReviewPage: React.FC<any> = (props: IReviewProps) =>
                 <Col md={6}>
                   <Row className="content-item">
                     <Label>Agency</Label>
-                    <AutoCompleteText
+                    <ParentSelect
```

Comment:
Use new agency components.

Reply:
Almost time to say goodbye to `AutoCompleteText.tsx` 😢